index.bs

```diff
@@ -68,32 +64,40 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMASCRIPT
 
 <em>This section is non-normative.</em>
 
+<div class="non-normative">
```

Comment:
This is the only use of this class in the spec. Should we have a separate issue to add more?

Reply:
I went through and added a bunch more. It caught some cases of using RFC 2119 keywords incorrectly. So, yay!